.packit.yaml

```diff
@@ -98,6 +99,7 @@ packages:
     upstream_tag_template: copr-cli-{version}
     actions:
       post-upstream-clone:
+        - sh -c "wait-for-copr --owner $COPR_OWNER --project $COPR_PROJECT python-copr-common `git rev-parse --short HEAD`"
```

Comment:
Shouldn't python-copr wait for python3-copr-common instead?

Reply:
yep, you are right